src/third-party/PhishingController.ts

```diff
@@ -168,8 +185,11 @@ export class PhishingController extends BaseController<
    * Temporarily marks a given origin as approved.
    *
    * @param origin - The origin to mark as approved.
+   * @returns Promise<void> when completed adding the origin to the whitelist
    */
-  bypass(origin: string) {
+  async bypass(origin: string) {
+    await this.fetchIfNecessary();
```

Comment:
It doesn't look like `bypass` relies upon the state of the phishing list, but rather configures the whitelist, so is it necessary to fetch here?

Reply:
Because it touches `this.state.whitelist` I just figured it might be necessary. This could very well be an oversight though, just trying to be cautious not to break anything by accident

Reply:
hmm yeah fair enough, even if whitelist hasn't been set yet, this code will just set the punycodeOrigin. 
```suggestion
```